compiler/pipes/gen-tree-postprocess.cpp

```diff
@@ -311,3 +320,55 @@ VertexPtr GenTreePostprocessPass::convert_array_with_spread_operators(VertexAdap
 
   return call;
 }
+
+VertexPtr GenTreePostprocessPass::convert_match(VertexAdaptor<op_match_proxy> match_vertex) {
```

Comment:
I think, this function needs a comment: what does it do? 

BTW, somewhere in comments you write "op_match", but actually, it's "op_match_proxy". Maybe, name it "op_match", why is it proxy? It's "match" (converted to op_switch later on, but it's a regular situation)